connect-inject/container_init.go

```diff
@@ -37,6 +38,8 @@ type initContainerCommandData struct {
 	PrometheusScrapePath string
 	// PrometheusBackendPort configures where the listener on Envoy will point to.
 	PrometheusBackendPort string
+	// EnvoyUID is the UID that `consul connect redirect-traffic` will use when tproxy is enabled.
```

Comment:
I found this comment confusing because it says it's envoy's UID but then it says it's for the command `consul connect redirect-traffic`.

Upon further reading I see that this is what gets passed into that command's `-proxy-uid` flag so that makes sense now but I'm thinking this comment should either just say "this is envoy's linux user id" without talking about the `redirect-traffic` command *or* it should say "this is envoy's linux user id that will be passed to the -proxy-uid flag of consul connect redirect-traffic".